src/main/kotlin/no/nav/familie/ks/sak/api/dto/VedtaksperiodeDto.kt

```diff
@@ -51,11 +53,13 @@ fun UtvidetVedtaksperiodeMedBegrunnelser.tilUtvidetVedtaksperiodeMedBegrunnelser
 data class VedtaksbegrunnelseDto(
     val begrunnelse: String,
     val vedtakBegrunnelseSpesifikasjon: String,
-    val begrunnelseType: BegrunnelseType
+    val begrunnelseType: BegrunnelseType,
+    val støtterFritekst: Boolean
 )
 
-fun Vedtaksbegrunnelse.tilRestVedtaksbegrunnelse() = VedtaksbegrunnelseDto(
+fun Vedtaksbegrunnelse.tilRestVedtaksbegrunnelse(sanityBegrunnelser: List<SanityBegrunnelse>) = VedtaksbegrunnelseDto(
```

Comment:
Gammel kode, men kunne vi ha tatt en namechange på denne samtidig til å reflektere  endringene fra rest -> dto?

```suggestion
fun Vedtaksbegrunnelse.tilVedtaksbegrunnelseDto(sanityBegrunnelser: List<SanityBegrunnelse>) = VedtaksbegrunnelseDto(
```

Reply:
Jepp! 😄